# A collection of barcode detections within an image.

std_msgs/Header                       header

ic_barcode_scanner/IcBarcodeResult[]  results
